extensions/amp-animation/0.1/web-animations.js

```diff
@@ -761,18 +761,37 @@ export class MeasureScanner extends Scanner {
           'Both "selector" and "target" are not allowed');
       targets = this.css_.queryElements(spec.selector);
       if (targets.length == 0) {
-        user().warn(TAG, `Target not found: "${spec.selector}"`);
+        const doc = this.css_.rootNode_;
```

Comment:
`this.css_.queryElements(spec.selector)` should always be correct as it is aware of shadow boundaries, so there might be a bug there or maybe wrong shadowRoot is being passed to the constructor. This should never need to pierce shadow boundaries.